### ISSUE TITLE
Not all spacing options work as expected for C# files

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#26996]
1.  Tools &gt; Options &gt; Text Editor &gt; C# &gt; Code Style &gt; Formatting &gt; Spacing
2.  Toggle the checkbox for &quot;Set other spacing options&quot; &gt; &quot;Insert space after cast&quot;
3.  Note that the code in the preview window doesn't change when the checkbox is toggled (it always behaves as if the checkbox is unchecked).  See attached screenshot.

This is not just a problem with the preview window.  It also affects the way Edit &gt; Advanced &gt; Format Document/Selection works.

Other spacing options -- but not all -- show the same behavior.  For instance, &quot;Insert spaces within parentheses of control flow statements&quot; works but &quot;Insert space within parentheses of expressions&quot; does not.

This is a regression, but I don't know how recent it is.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/237798/not-all-spacing-options-work-as-expected-for-c-fil.html
VSTS ticketId: 604163_
_These are the original issue comments:_
(no comments)
_These are the original issue solutions:_
(no solutions)

**Comment by jcouv** [comment]
Tagging @jinujoseph as this sounds like a possible regression.

**Comment by Neme12** [comment]
Just a question: I'm getting Access Denied on the link to developer community. Is this expected? I'm still able to access other feedback items, so it seems to be related to this particular one. Are some feedback items private to Microsoft?

**Comment by sharwell** [comment]
> Are some feedback items private to Microsoft?

Yes, but it doesn't readily tell us which is which. In this case, the feedback item has no comments and the text is reproduced verbatim above.

**Comment by Neme12** [comment]
Thanks. I usually click the link to see the version used because it's not copied to the issue.

**Comment by jmarolf** [comment]
@Neme12 yeah, that's another reason you can't access it.  It could be a version number for a VS that isn't released yet.  In this case though it was 15.7.1 though.

**Comment by Neme12** [comment]
Are the internal version numbers secret? I see them very often in issues created by team members, for example: #27010

**Comment by jmarolf** [comment]
@Neme12 I mean  the roslyn team doesn't care in most cases (surprise there is going to be a VS 15.8 someday! 😃 ) but the devdiv internal processes are more conservative about this sort of thing.
